frontend/src/components/Wheel/index.js

```diff
@@ -19,46 +46,28 @@ const Wheel = () => {
 
   return (
     <div>
-      <table>
-        <tr>
-          <th>ID</th>
-          <th>Name</th>
-          <th>Logo</th>
-          <th>Prizes</th>
-          <th>Color1</th>
-          <th>Color2</th>
-        </tr>
-
-        <tr key={wheelInfo?.id}>
-          <td>{wheelInfo?.id}</td>
-          <td>{wheelInfo?.name}</td>
-          <td>{wheelInfo?.logo}</td>
-
-          <td>
-            {wheelInfo?.prizes?.map((prize) => {
-              return (
-                <div key={prize.id}>
-                  <span>Name: {prize.name}</span>
-                  <span> Percentage: {prize.percentage}</span>
-                  <span> Image: {prize.image}</span>
-                </div>
-              );
-            })}
-          </td>
-
-          <td>{wheelInfo?.config?.main_color}</td>
-          <td>{wheelInfo?.config?.secondary_color}</td>
-        </tr>
-      </table>
-
-      <div className={styles.align}>
-        <SpinButton
-          colors={[
-            wheelInfo?.config?.main_color,
-            wheelInfo?.config?.secondaryColor,
-          ]}
-        />
-      </div>
+      <h1 className={styles.center}>{wheelInfo.name}</h1>
+      <div className={styles.arrow}></div>
+      <ul className={rotate}>
+        {wheelInfo?.prizes?.map((prize, i) => {
+          return (
+            <li className={styles.prize} key={prize.id} style={wheelStyle(i)}>
```

Comment:
This isn't a list if every list only has one item inside. Do you mean moving `<li>` before `wheelInfo.prizes.map`?

Reply:
We changed that to prizesInfo.map((prize, index), and that's a list with multiple objects (prizes). Aren't we supposed to have each one of them being an item?